tests/asn1_test.cc

```diff
@@ -0,0 +1,62 @@
+#include <gtest/gtest.h>
+
+#include <string>
+
+#include <boost/smart_ptr.hpp>
+
+#include <SecondaryConfig.h>
+#include "asn1/helpers.h"
+
+using namespace Uptane::ASN1;
+
+TEST(asn1, serialize_simple) {
+  SecondaryConfig *secConf = static_cast<SecondaryConfig *>(calloc(1, sizeof(*secConf)));
```

Comment:
Why not: 
` SecondaryConfig *secConf = new SecondaryConfig;
`

Reply:
Hm, I assume SecondaryConfig is ASN1::SecondaryConfig. Probable better to use  `using namespace Uptane;
` and `ASN1::SecondaryConfig`

Reply:
Can't use `new` because the deallocation is handled by the asn1c generated code which is in C and uses `free`. We most probably want to stick with their version as it frees the objects recursively.

Reply:
About the namespace comment: you're right, I forgot to update the test case to the new style of code organization I used.